scripts/profiling_runner.bash

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/bash
```

Comment:
Can you please add how to use this and the plot scripts at the end of the README?

Reply:
Moved it to the end :heavy_check_mark: